daisy_workflows/build-publish/rhui/health_checks/health_check.py

```diff
@@ -0,0 +1,86 @@
+#  Copyright 2022 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Health check for RHUI infrastructure nodes."""
+
+import argparse
+import logging
+import os
+import sys
+import typing
+
+
+def nfs_is_read_mounted(logger: logging.Logger):
+  nfs_mount = "/var/lib/rhui/remote_share"
+  logger.info("checking that %s is mounted", nfs_mount)
+  os.statvfs(nfs_mount)
```

Comment:
this will just ensure the mount *point* exists, not that it is mounted. i think what we want here is really to check /proc/self/mounts, idk if there's an existing python function for it (there may be, i don't know)